assets/src/edit-story/components/panels/textStyle/useRichTextFormatting.js

```diff
@@ -112,19 +114,24 @@ function useRichTextFormatting(selectedElements, pushUpdate) {
         handleClickBold: () => selectionActions.toggleBoldInSelection(),
         // All these keep their arguments:
         handleSelectFontWeight: selectionActions.setFontWeightInSelection,
+        handleResetFontWeight: (weight) => {
+          push(htmlFormatters.setFontWeight, weight);
+        },
```

Comment:
If this is the same function in both returns, it should probably be defined above only once.